config/migrations/20230419160225-add-report-class.sparql

```diff
@@ -0,0 +1,50 @@
+PREFIX dossier: <https://data.vlaanderen.be/ns/dossier#>
+PREFIX besluitvorming: <https://data.vlaanderen.be/ns/besluitvorming#>
+
+INSERT {
+    GRAPH <http://mu.semte.ch/graphs/organizations/minister> {
+        ?stuk a besluitvorming:Verslag .
```

Comment:
@SchrodingersCat00 During https change of besluitvorming I changed this, used to be `dossier:Verslag` but that doesn't match the type you added.
Is this correct?

Reply:
It should be `besluitvorming:Verslag` instead of `dossier:Verslag` so this change is correct.